catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/pom.xml

```diff
@@ -0,0 +1,122 @@
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>registry-admin-modules</artifactId>
+        <groupId>org.codice.ddf.registry</groupId>
+        <version>2.9.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>registry-admin-remote-ui</artifactId>
+    <packaging>bundle</packaging>
+    <modelVersion>4.0.0</modelVersion>
+    <name>DDF :: Registry :: Admin :: Remote UI</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>ddf.admin.core</groupId>
+            <artifactId>admin-core-appservice</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+                <includes>
+                    <include>**/*.xml</include>
+                </includes>
+            </resource>
+        </resources>
+        <plugins>
+            <plugin>
+                <groupId>com.github.eirslett</groupId>
+                <artifactId>frontend-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>grunt install</id>
+                        <goals>
+                            <goal>grunt</goal>
+                        </goals>
+                        <phase>install</phase>
+                        <configuration>
+                            <arguments>build</arguments>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <configuration>
+                    <instructions>
+                        <!--we will use this servlet filter once it has been made generic to protect this app-->
+                        <!--<Embed-Dependency>search-securityservletfilter</Embed-Dependency>-->
+                        <Web-ContextPath>/admin/registries</Web-ContextPath>
```

Comment:
just be aware that if you do change this you will have to change your relative paths in main.js